test/integration/buoyancy.cc

```diff
@@ -42,6 +42,121 @@ class BuoyancyTest : public InternalFixture<::testing::Test>
 {
 };
 
+/////////////////////////////////////////////////
+TEST_F(BuoyancyTest, IGN_UTILS_TEST_DISABLED_ON_WIN32(RestoringMoments))
```

Comment:
Sweet, I think this is the first test where we actually start 2 separate servers and compare results, great job!